backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationExtractor.java

```diff
@@ -25,35 +33,88 @@ public OrganizationExtractor(AuthorizationProperties properties) {
     this.properties = properties;
   }
 
-  @Override
-  public List<OrganizationRoleClaims> convert(Collection<? extends GrantedAuthority> source) {
+  /**
+   * There are two kinds of claims we care about - role claims and facility claims. Role claims take
+   * the format SR-<ENV>-TENANT:<ORG_EXTERNAL_ID>:<ROLE> Facility claims take the format
+   * SR-<ENV>-TENANT:<ORG_EXTERNAL_ID>:FACILITY_ACCESS:<FACILITY_UUID>
```

Comment:
it all originates from the JWT -- this is no less secure than the existing OrganizationRoleClaims parsing logic which accepts your roles from the JWT

how would you suggest enforcing? it would have to limit the possible values of a statically defined enum.

Reply:
lol i should've been more vocal that the parsing mechanics were my concern, but frankly there are different issues regardless of how you structure it. i'm not dying to redo it, but if there's a time to, it's now.